### PR TITLE
SPCTR-1910 Fix: Z-Index extension is not working for the dynamic blocks.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1910

### Screenshots

- Screencast - https://share.getcloudapp.com/6quGo7bn
- Screenshot - https://share.getcloudapp.com/OAujZEyR

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — APPROVED** [review]
Once Image Gallery, Counter, Login, Register blocks are merged, we can probably even create a PHP helper function for z-index functionality

File: README.md [hunk 1]
```diff
@@ -206,6 +206,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Container - Does not work properly when it is used inside cover block.
 * Fix: Table Of Contents - Page was not scrolling to the headings on the frontend for some edge cases.
 * Fix: Post Carousel - On reducing or increasing Post per page all items sticks to each other in editor.
+* Fix: Z-index extension - Was not working for dynamic block.
```
Comment (RajkiranBagal) [inline README.md:209]:
```suggestion
* Fix: Z-Index extension is not working for the dynamic blocks.
```

File: blocks-config/lottie/class-uagb-lottie.php [hunk 2]
```diff
@@ -169,6 +169,31 @@ public function render_html( $attributes ) {
 				$mob_class = ( isset( $attributes['UAGHideMob'] ) ) ? 'uag-hide-mob' : '';
 			}
 
+			$zindex_desktop             = '';
+			$zindex_tablet              = '';
+			$zindex_mobile              = '';
+			$zindex_wrap                = array();
+			$uagb_common_selector_class = '';
+
+			if ( array_key_exists( 'zIndex', $attributes ) || array_key_exists( 'zIndexTablet', $attributes ) || array_key_exists( 'zIndexMobile', $attributes ) ) {
```
Comment (RajkiranBagal) [inline blocks-config/lottie/class-uagb-lottie.php:178]: This is a very un optimized may of doing this logic. Please improve this. Avoid unnecessary function calls & condition checks.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: blocks-config/lottie/class-uagb-lottie.php (hunk 2 above)
Comment (AbhijitNage123) [reply blocks-config/lottie/class-uagb-lottie.php:178]: I removed unnecessary condition checks from the variable. Please re-check.

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**